parser_library/src/processing/op_code.cpp

```diff
@@ -18,45 +18,47 @@
 
 namespace hlasm_plugin::parser_library::processing {
 
-inline unsigned char get_reladdr_bitmask(context::id_index id)
-{
-    if (id.empty())
-        return 0;
-
-    if (auto p_instr = context::instruction::find_machine_instructions(id.to_string_view()))
-        return p_instr->reladdr_mask().mask();
+constexpr const auto common_processing_status_cache_key_details = std::pair((unsigned char)1, (unsigned char)0);
 
-    if (auto p_mnemo = context::instruction::find_mnemonic_codes(id.to_string_view()))
-        return p_mnemo->reladdr_mask().mask();
+// (value of L'* expression, reladdr mask)
+inline std::pair<unsigned char, unsigned char> get_processing_status_cache_key_details(std::string_view id) noexcept
+{
+    if (!id.empty())
+    {
+        const auto [mi, mn] = context::instruction::find_machine_instruction_or_mnemonic(id);
 
-    return 0;
+        if (mn)
+            return std::pair(static_cast<unsigned char>(mi->size_in_bits() / 8), mn->reladdr_mask().mask());
```

Comment:
Just for the sake of consistency?

```suggestion
            return std::pair(static_cast<unsigned char>(mn->size_in_bits() / 8), mn->reladdr_mask().mask());
```

Reply:
Only the base machine_instruction struct has the size info available.